pkg/collectors/devices/ptp_devices.go

```diff
@@ -12,77 +13,173 @@ import (
 )
 
 type PTPDeviceInfo struct {
-	VendorID string `json:"vendorId"`
-	DeviceID string `json:"deviceInfo"`
-	GNSSDev  string `json:"GNSSDev"` //nolint:tagliatelle // Because GNSS is an ancronym
+	Timestamp string `json:"date"`
+	VendorID  string `json:"vendorId"`
+	DeviceID  string `json:"deviceInfo"`
+	GNSSDev   string `json:"GNSSDev"` //nolint:tagliatelle // Because GNSS is an ancronym
 }
 
 type DevDPLLInfo struct {
+	Timestamp string `json:"date"`
 	EECState  string `json:"EECState"`  //nolint:tagliatelle // Because EEC is dpll name
 	PPSState  string `json:"PPSState"`  //nolint:tagliatelle // Because PPS is dpll name
 	PPSOffset string `json:"PPSOffset"` //nolint:tagliatelle // Because PPS is dpll name
 }
 type GNSSDevLines struct {
-	Dev   string `json:"dev"`
-	Lines string `json:"lines"`
+	Timestamp string `json:"date"`
+	Dev       string `json:"dev"`
+	Lines     string `json:"lines"`
 }
 
 func GetPTPDeviceInfo(interfaceName string, ctx clients.ContainerContext) (devInfo PTPDeviceInfo) {
 	// Find the dev for the GNSS for this interface
-	gnssDev := commandWithPostprocessFunc(ctx, strings.TrimSpace, []string{
+	cmds := &clients.CmdGroup{}
```

Comment:
This looks like it would be better with a function to build and populate the command group from a minimal set of inputs (`(command text, optional? cleanup func)`)

Reply:
Ignore this, sorted in a later commit